Pd 727 the glyph buttons sketch library doesn't support multiple sizes

sooachung opened the pull request:
Yay, my first pull request!
Added Large, Extra Large size for: Static, Light/ Disabled, Dark/ Disabled buttons.

sooachung committed:
Added some test squares

Added a total of 6 test squares

sooachung committed:
Added Large, Extra Large buttons

Added Large and Extra Large for:
Static, Light/ Disabled, Dark/ Disabled

sooachung committed:
Removed the test squares on the reference

audroue commented:
Awesome! I'll look over it :)

audroue reviewed (commented):
Great first start!

Couple of things I noticed:

- On your last commit/push save the file on the cover page so that in previews we see the pretty preview screen 

![image](https://user-images.githubusercontent.com/4782672/87592394-c84c9b80-c6b7-11ea-8c93-ffcb4a16cd61.png)

- The reference page still needs to get updated to show the new symbols you made

![image](https://user-images.githubusercontent.com/4782672/87592443-dc909880-c6b7-11ea-978b-0b8730a555ca.png)

- This is a bit of scope creep, but do you think you could remove the other focus and hover states, since we no longer include these in our library (but we do on our specs for Eng).

![image](https://user-images.githubusercontent.com/4782672/87592650-3abd7b80-c6b8-11ea-90bc-b842af79ed8f.png)

- The elements in the new symbols don't seem to be pinned or fixed in size. Make sure the new symbols reflect the pinning/fixing that the older symbols have so that on resize these symbols retain their constraints — we can go more in detail over this if you want! 

![image](https://user-images.githubusercontent.com/4782672/87592829-81ab7100-c6b8-11ea-923b-bc79578defd5.png)

- There seem to be some elements/symbols that are copied on top of each other. Additionally, the overrides panel is super long. Compare it to the older symbols to see how long and what the end user should be able to edit:

![image](https://user-images.githubusercontent.com/4782672/87592964-b5869680-c6b8-11ea-8b16-5c7d0c77bfb6.png)

Great first pass! Continue the great work 🥳

sooachung committed:
Updates based on Rob's comment

5 changes were made

audroue commented:
Heya here's some feedback!

I'd organize it with light & dark mode at the top level. So like this **dark/light mode > size > state**.

![image](https://user-images.githubusercontent.com/4782672/88220070-8039f580-cc30-11ea-9028-969f6532b0cb.png)

Also I'm noticing some of the glyph button icons are still getting distorted on resize:

![image](https://user-images.githubusercontent.com/4782672/88220213-aa8bb300-cc30-11ea-99da-ebdc091f6f56.png)

The new larger glyph buttons don't allow me to switch the icon within them. Below is an example that shows how switching the icon works in the overrides panel. I think this may be happening because you didn't insert the symbol in the reference symbol but instead copy pasted the the elements shape there instead.

![image](https://user-images.githubusercontent.com/4782672/88220457-fe969780-cc30-11ea-856f-048171f674ff.png)

Would rename the reference layer to simply "reference"

![image](https://user-images.githubusercontent.com/4782672/88220792-795fb280-cc31-11ea-9dc8-dd60090c4b41.png)

Almost there! Otherwise looks good.

sooachung committed:
Update Glyph Buttons [LeafyGreen].sketch

Organzied the levels of state, made sure the glyph buttons are not getting distorted by inserting it as a symbol instead of copying it. Removed # infront of reference.

audroue reviewed (approved):
All looks good!
